intranet/apps/eighth/views/admin/blocks.py

```diff
@@ -57,8 +60,10 @@ def add_block_view(request):
                 if not ltr:
                     continue
                 if ltr not in current_letters:
-                    EighthBlock.objects.create(date=fmtdate, block_letter=ltr)
-                    messages.success(request, "Successfully added {} Block on {}".format(ltr, fmtdate))
+                    EighthBlock.objects.create(date=fmtdate, block_letter=ltr, signup_time=signup_time)
+                    messages.success(
+                        request, "Successfully added {} Block on {} with signups shown to students as closing at {}".format(ltr, fmtdate, signup_time)
+                    )
             for ltr in current_letters:
                 if not ltr:
                     continue
```

Comment:
You should add something like
```
else:
    blk = EighthBlock.objects.get(date=fmtdate, block_letter=ltr)
    blk.signup_time = signup_time
    blk.save()
    messages.success(request, "Successfully changed the signup time on {} Block on {}".format(ltr, fmtdate))
   ```
   under the if statement in line 70. This will allow the signup time to be changed.